exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -202,7 +203,7 @@ class OTLPExporterMixin(
     def __init__(
         self,
         endpoint: Optional[str] = None,
-        insecure: Optional[bool] = None,
+        insecure: Optional[bool] = False,
```

Comment:
Hmm, now I think about it, will this be a breaking change?

Reply:
I have reverted this so that if it is not set, and the URL scheme is `http`, we infer `insecure=True`.